build.gradle

```diff
@@ -63,6 +63,9 @@ dependencies {
     provided "org.grails:grails-dependencies"
     provided 'javax.servlet:javax.servlet-api:3.1.0'
 
+    compile "org.grails:grails-core"
+    compile "org.grails:grails-dependencies"
```

Comment:
Shouldn't depend on `grails-dependencies` directly, as that will bind the plugin to all web related plugins.